flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamGraphGenerator.java

```diff
@@ -131,11 +130,14 @@
 
 	private String jobName = DEFAULT_JOB_NAME;
 
-	private GlobalDataExchangeMode globalDataExchangeMode = GlobalDataExchangeMode.ALL_EDGES_PIPELINED;
+	private GlobalDataExchangeMode globalDataExchangeMode = null;
```

Comment:
Same here: I would remove the explicit `null` from here. But feel free to leave it if you prefer this.